docs/content/flink/sql-write.md

```diff
@@ -261,8 +261,26 @@ CREATE TABLE my_partitioned_table (
     'partition.time-interval'='1 d',
     'partition.idle-time-to-done'='15 m',
     'partition.mark-done-action'='done-partition'
+    -- You can also customize a PartitionMarkDoneAction to mark the partition completed.
```

Comment:
Please provide a detailed description of the business scenario in this pr's Purpose.

Reply:
Refer to Flink's partition-commit policy.  I has implemented a custom partition-commit policy in my company, and I will also contribute it to the community in the future.

Reply:
ok